src/main/java/com/andersen/hw/model/Ticket.java

```diff
@@ -1,29 +1,45 @@
 package com.andersen.hw.model;
 
 import com.andersen.hw.enums.SectorType;
+import com.andersen.hw.enums.TicketType;
 
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.Currency;
 import java.util.Locale;
-
 import java.util.Objects;
 
 public class Ticket implements Printable, Identifiable {
-    private final int classId;
-    private final String ticketId;
-    private final String concertHall;
-    private final String eventCode;
+    private int classId;
+    private String ticketId;
```

Comment:
the same: create just 1 autogenerated id for each instance of Ticket

Reply:
done